openpype/modules/kitsu/utils/sync_service.py

```diff
@@ -119,17 +124,16 @@ def _update_project(self, data):
 
         # Write into DB
         if update_project:
-            self.dbcon = self.dbcon.database[project_name]
+            self.dbcon.Session["AVALON_PROJECT"] = project_name
             self.dbcon.bulk_write([update_project])
 
     def _delete_project(self, data):
         """Delete project."""
-        project_doc = self.dbcon.find_one(
```

Comment:
You actually have to query from kitsu in all cases because you have to get the project name from there. Without project name you can't query project document by zou id because you don't know in which collection it should be found.